src/www/pages/patterns/header-and-footer/index.njk

```diff
@@ -164,12 +156,74 @@
 
 {# Full footer #}
```

Comment:
Can you move the full variant of the footer so that is above the mobile-app variant. Thanks.